.env.example

```diff
@@ -22,3 +22,7 @@ QUEUE_DRIVER=sync
 # Filesystems
 FILESYSTEM_DRIVER=local
 DEPENDENCY_PUBLIC_FOLDER=dependencies
+
+# External Dependencies
+BUGSNAG_API_KEY=
+BUGSNAG_LOGGER_LEVEL=WARNING
```

Comment:
Newline at EOF? :)

Reply:
Wasn't included before, not included by default?
https://github.com/laravel/laravel/blob/master/.env.example

Reply:
Maybe just me, but fine, it's not a biggie!